docs/azure/sdk/unit-testing-mocking.md

```diff
@@ -231,6 +231,36 @@ var finder = new AboutToExpireSecretFinder(TimeSpan.FromDays(2), secretClient);
 
 This approach is useful when you would like to consolidate the dependency creation and share the client between multiple consuming classes.
 
+## Understand Azure Resource Manager clients
```

Comment:
```suggestion
## Understand Azure Resource Manager (ARM) clients
```
Maybe unnecessary but ARM seems to be the commonly used term and I'm not confident that customers know the long form of the acronym.

Reply:
I also think we should use the ARM acronym here